functions/mult.h

```diff
@@ -0,0 +1,28 @@
+#ifndef FUNCTIONS_MULT_H
+#define FUNCTIONS_MULT_H
+
+// Usage: Pct<F, V>
```

Comment:
Documentation is for the Pct<> function.
Abbreviations should be avoided unless they are completely obvious, so I think Percentage should be the right thing here.

Reply:
resolved